### PR TITLE
Get Media Attributes

### PR DESCRIPTION
**Pull request by johndelong**
This update will enable the library to determine what type of mediaContent an RSSItem contains. Also puts mediaContent into a dictionary.

### CODE REVIEW

File: RSSParser/RSSItem.h [hunk 1]
```diff
@@ -10,6 +10,12 @@
 
 @interface RSSItem : NSObject <NSCoding>
 
+typedef enum {
+    video,
+    audio,
+    unknown
```
Comment (joshfriend) [inline RSSParser/RSSItem.h:17]: what about image?

File: RSSParser/RSSItem.h [hunk 2]
```diff
@@ -10,6 +10,12 @@
 
 @interface RSSItem : NSObject <NSCoding>
 
+typedef enum {
```
Comment (jkloo) [inline RSSParser/RSSItem.h]: `NS_ENUM` is the recommended means of creating an enum http://nshipster.com/ns_enum-ns_options/

File: RSSParser/RSSItem.h [hunk 3]
```diff
@@ -10,6 +10,12 @@
...
+    video,
+    audio,
+    unknown
+} contentType;
```
Comment (jkloo) [inline RSSParser/RSSItem.h]: "types" should be UpperCamelCase

File: RSSParser/RSSItem.h (hunk 2 above)
Comment (carrchr) [inline RSSParser/RSSItem.h]: ^^Yes^^

**Pull request merged**